rust/src/transformations/make_stable_lazyframe/h_stack/test.rs

```diff
@@ -40,6 +40,9 @@ fn test_with_column() -> Fallible<()> {
 // LazyFrame::fill_nan works for "free", because it breaks down to pre-existing primitives:
 //    hstack with expressions that use fill_nan and alias
 #[test]
+// ignored because Polars 1.0.0 changes behavior of fill_nan to de-sugar into .with_columns(all().fill_nan(v)),
+// and we don't support all() at this time
+#[ignore]
```

Comment:
Maybe just drop the test, then? Or if it should stay, add TODO with a link to the issue that will resolve this?